packages/appium/docs/en/ecosystem/index.md

```diff
@@ -22,6 +22,7 @@ Intro](../intro/drivers.md)
 
 |Driver|Installation Key|Platform(s)|Mode(s)|
 |--|--|--|--|
+|[Chromium](https://github.com/appium/appium-chromium-driver)|`chromium`|macOS, Windows, Linux|Web|
```

Comment:
Android as well?
https://github.com/appium/appium-chromium-driver/blob/ac193e2fde0e1c19f25261e56939ea11434c32dd/package.json#L71-L75

Reply:
oh, yeah i don't really want to support android via this driver, it would require a bunch of other stuff to make ergonomic when we already have it in the uiauto2 driver. i'll remove android from the driver.

Reply:
yea, makes sense